models/articles.models.js

```diff
@@ -14,18 +14,64 @@ exports.selectArticle = (articleId) => {
     });
 };
 
-exports.selectArticles = () => {
+exports.selectArticles = (topic, sortBy, orderBy) => {
+  if (
```

Comment:
this long if && if && could be simplified into an array of possible choices and check wether it is in the array, this could make it easier to change if we ever decided to